bash_scripts/bash_script/script.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+function dirsolder {
+        logger "dirs older"
+        IFS=$'\n'
+        read -p "Input dir's older than: " days
+        date=$(date +"%b %d %H:%M" -d "$days day ago")
+        for var in $(ls -ld ~/*/  | awk '{printf "%s %s %s %s\n", $6, $7, $8, $9}')
```

Comment:
Better to have a directory as a variable to be more flexible.

Reply:
Use dir's as variables